cross_compile/sysroot_compiler.py

```diff
@@ -161,7 +161,8 @@ def __init__(
       ros_workspace_dir: str,
       platform: Platform,
       docker_config: DockerConfig,
-      custom_setup_script_path: Optional[str],
+      custom_setup_script_path: Optional[str] = None,
+      custom_data_dir: Optional[str] = None,
```

Comment:
`custom_data_dir_path`

Reply:
However, I suggest just making it suffixed with `dir` or `str` to avoid confusion between `path` from `pathlib` and a regular string.

Reply:
I'm removing `path`, it's redundant to `dir`